.github/workflows/testing-and-deployment.yml

```
@@ -180,6 +180,10 @@ jobs:
         if: ${{ matrix.vtk-version == '9.0.3' }}
         run: pip install 'numpy<1.24'
 
+      - name: Limit Matplotlib for VTK<=9.2.2
+        if: ${{ matrix.vtk-version != 'latest' }}
```

I am not familiar with GitHub Actions syntax. Is this the correct wording for small versions?
```suggestion
        if: ${{ matrix.vtk-version <= '9.2.2' }}
```

---

AFAIK, this won't work as we're doing literal string comparison here

---

OK. If you don't mind please use this.
```suggestion
        if: ${{ matrix.vtk-version != '9.2.2' }}
```

---

OK. I got As is is the best choice.

---

Yeah, its not great but its what works for the current CI configuration. When we update the VTK versions we test against, this will have to be updated.